src/TestFramework/TestFramework.Extensions.Csv/TestFramework.Extensions.Csv.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- TODO: (before merge) Is this hack relevant for a new project? -->
```

Comment:
Good question, it's probably fine as this is (was?) mainly for some VS binding redirects.

Reply:
@Evangelink by fine, do you mean it can be removed for the new projects? or do you mean just keep as-is?